numalogic/udfs/inference.py

```diff
@@ -31,7 +31,7 @@
 LOCAL_CACHE_TTL = int(os.getenv("LOCAL_CACHE_TTL", "3600"))
 LOCAL_CACHE_SIZE = int(os.getenv("LOCAL_CACHE_SIZE", "10000"))
 LOAD_LATEST = os.getenv("LOAD_LATEST", "false").lower() == "true"
-METRICS_ENABLED = bool(int(os.getenv("METRICS_ENABLED", default="1")))
+METRICS_ENABLED = bool(int(os.getenv("METRICS_ENABLED", default="0")))
```

Comment:
do we have to initialize this again in each UDF? doesn't main.py do this?

Reply:
Discussed this offline, we are seeing cyclic dependencies and decided to do this after refactoring.